README.md

```diff
@@ -18,7 +18,8 @@ to localized information (which is not shared) or additional information (that c
 
 - [clusters/android.json](clusters/android.json) - Android malware galaxy based on multiple open sources.
 - [clusters/banker.json](clusters/banker.json) - A list of banker malware.
-- [clusters/botnet.json](clusters/botnet.json) - A list of known botnets.
+- [clusters/stealer.json](clusters/stealer.json) - A list of malware stealer.
+- [clusters/banker.json](clusters/banker.json) - A list of banker malware.
```

Comment:
This line should be the link to the botnet file, instead of a duplicate of banker)